### ISSUE TITLE
[BUG]:  Android Build

### ISSUE THREAD

**Issue by ekrenzin** [Unity-Technologies/com.unity.webrtc#767]
### Package version

2.4.0-exp.7

### Environment

```markdown
* OS: Windows 11 / Oculus Quest 2
* Unity version: 2021.3.5f
```

### Steps To Reproduce

1. Build & run apk
2. create a new peer with `new RTCPeerConnection(ref config);`

### Current Behavior

Error when creating peer:

`NullReferenceException: Object reference not set to an instance of an object
17:32:29.773
Unity
at Unity.WebRTC.RTCPeerConnection..ctor (Unity.WebRTC.RTCConfiguration& configuration) [0x0002d]`

### Expected Behavior

When building for windows I am able to connect to the remote peer and receive audio / video. The exact same code built for android (oculus quest 2) has the error. Other web requests still work on the android build, so I do not suspect web permissions.

### Anything else?

Possibly related to issue: https://github.com/Unity-Technologies/com.unity.webrtc/issues/411

**Comment by karasusan** [comment]
@ekrenzin 
Can I have the Unity project you testing?

**Comment by ekrenzin** [comment]
> @ekrenzin Can I have the Unity project you testing?

@karasusan I was able to resolve the issue, unexpectedly, the issue appears to have been caused by my target architectures. 

I just finished creating a new test project to share so we could debug this, and while doing so I doubled checked all of my build settings on my original project. I realized that I was also targeting ARMv7.

Because the error message was pointing to the configuration, I was looking in the wrong place. 

For anyone experiencing this issue, check your build settings to
![image](https://user-images.githubusercontent.com/33398104/177478716-c9e684f0-baad-4e1f-90fb-016296501ad0.png)
